### ISSUE TITLE
Second initialization fails

### ISSUE THREAD

**Issue by kamil7x** [videojs/videojs-vimeo#68]
Hi

I have a problem with ajax-loaded videos. When i am loading bunch of videos, I am initializing them using videojs function.

For each element (using jQuery.each()):

```
var newId = 'video-id-'+idCount;
var setup = JSON.parse(el.attr("data-setup"));
el.removeAttr( "data-setup" );
videojs(newId, setup, function() {});
idCount++
```

And this is working for Youtube videos.
But for video from Vimeo, after first videos are loaded, when I click Load more (it gets new HTML from server and runs code, I wrote above on this HTML), I have an error in console:

```
TypeError: el is undefined
```

After some research, I found that Dom.createEl() fails to create element.
<div> with id is created, but without content.

In short:
1. When I am initializing one element, it is working
2. When I am initializing more elements at once (in one jQuery.each() loop) - it is working
3. When I am trying to initialize additional elements, in second run (no matter if I am running videojs() from console to test, or from another jQuery.each() loop) - it fails

What can I do with this?

**Comment by Magiweb** [comment]
@kamil7x I think your issue might be related to the issue I raised here: https://github.com/eXon/videojs-vimeo/issues/65. It's similar because when I try to dynamically create an element and initialise VideoJS using this plugin it breaks. Also switching / adding sources dynamically appears to be an issue.

@hendrathings has pushed an update a few days ago, but it doesn't seem to be merged yet, although @eXon seems to have done an update, so I'm pretty confused right now as to whether or not to use the update on the master branch or to try hendrathings branch instead (since it's not merged yet). 

Hopefully they'll let us know :)

**Comment by pikilon** [comment]
+1

**Comment by kamil7x** [comment]
This issue is very old, and if I remeber correctly, update worked. I am closing this issue.
